nlp/environment.yml

```diff
@@ -1,9 +1,18 @@
 name: birdy
```

Comment:
note on this file's review: I see that it differs from the one we have here : https://www.crim.ca/stash/projects/DACCS/repos/wp15/browse/environment.yml

I'm not certain about the dependencies of this package, like `torch==1.8.1` that has been removed, and others

Reply:
I trimmed down the environmnet to have as less dependecies as possible, as long as the python files would run and so I kept in mind the we would add all the other dependencies after. So should I add the missing dependencies from here : 
https://www.crim.ca/stash/projects/DACCS/repos/wp15/browse/environment.yml ?

Reply:
@ahandan I'm not aware of the dependencies that are actually required. @ChaamC Could you advise on this one?

Reply:
I am not sure either which dependencies are required for the nl2query code added here, as I am not familiar with the wp15 repo.

The nl2query folder is just one of multiple folders found on the wp15 repo, so I'm guessing it's possible that the dependencies found on the environment.yml file of the wp15 repo are not all necessarily required. It is also be a good idea to trim down the dependencies to the minimum to minimize the size of the Docker image.

Are we really sure that every files included here are working?

Maybe another idea would be to check where the dependencies are used (cloning the project wp15 and ctrl+f search?)
If I check the netCDF4 and threddsclient for example, they seem only to be used in a .py file outside of the nl2query folder, so probably not required here...

Reply:
@ahandan Ok then, you approach of trimming down dependencies make sense. We will adjust later on if there are missing deps.